package Exception;

public class Use_of_throws {
	public static void main(String[]args) throws Throwable {
			System.out.println("hii");
			Thread.sleep(2000);
		}
	}


